Justfile

```diff
@@ -140,6 +141,15 @@ test-all: test-pkgdb impure-tests integ-tests functional-tests
 @pkgdb +args="": build-pkgdb
     pkgdb/bin/pkgdb {{args}}
 
+
+# ---------------------------------------------------------------------------- #
+
+# Clean ( remove ) built artifacts
+@clean:
+    pushd cli; cargo clean;
+    make -C pkgdb clean;
```

Comment:
Is this like make where each line is a new shell in the original path? (.ONESHELL ?)

Reply:
Yes